### DISCUSSION TITLE
Peakeater problems.

### DISCUSSION THREAD

**Discussion by coachz1** [vvvar/PeakEater#75] [Q&A]
I tried Peakeater on bass guitar and found it to require 1.9dB of gain just to be the same volume with ceiling at 0 and input at 0 and oversampling on 16.  Also in Reaper the Delta with no limiting shows it removing lots of mids and mid highs.  With no limiting activated it should not be removing anything.  Thanks for any ideas.

**Comment by joetink** [comment]
I have noticed this issue, as well.  I add about 1.2db to get the track back to proper gain after simply inserting the plugin.

**Comment by coachz1** [comment]
The author seems to have disappeared regarding addressing issues sadly.

**Comment by joetink** [comment]
If that's the case, it's a shame.  This plugin, with just a few tweaks, could be the best free clipper by a large margin.

**Comment by molokoshka** [comment]
Hello! 
PeakEater v0.8.2 and lower version don't work at my mac. VST3 and AU both don't work 😭
I use _MacBook Pro 15 2018 macOS Mojave 10.14.6 (18G103) Intel_. It don't work in Logic 10.4.4 / Ableton Live 11 / FL Studio 20. I try to use command in terminal but it don't want to start. GateKeeper disabled. 

`**sudo xattr -cr** 

**sudo xattr -r -d com.apple.quarantine** 

**sudo codesign --force --deep --sign -** 

**sudo chmod -R 777**`

If anyone has encountered this problem, can you please help me how to open it?
I attach screenshots maybe it can help. :((((((((

<img width="1680" alt="Screenshot 2023-11-28 at 6 05 08 AM" src="https://github.com/vvvar/PeakEater/assets/152241009/2469a29e-4eb6-40fd-aaa4-ba4232cbb84a">
<img width="414" alt="Screenshot 2023-11-28 at 6 05 46 AM" src="https://github.com/vvvar/PeakEater/assets/152241009/c76b0a35-345f-414f-a1a0-a282ae029185">
<img width="1680" alt="Screenshot 2023-11-28 at 6 06 17 AM" src="https://github.com/vvvar/PeakEater/assets/152241009/e1c51b77-c71c-4748-a906-d8d8cab47584">

**Reply by molokoshka** [reply]
FIXED BY UPDATING TO MONTEREY *

**Reply by maxgorbushin** [reply]
I have the same problem! Help me please! peakeater not open in any DAW on OSX 10.14. What i must to do?
